src/main/webapp/feedback/feedback.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+window.iconOnClick = iconOnClick;
+
+/**
+ * Sends `iconType`, video time stamp of when icon was clicked,
+ * and lecture ID to be stored in database.
+ */
+function iconOnClick(iconType) {
+  const videoTimeStamp = window.videoPlayer.getCurrentTime();
+  const url = new URL('/icon-feedback', window.location.origin);
```

Comment:
can you update the strings to be constants?

Reply:
Might be a good idea to bundle these constants and function in some sort of `Feedback` class

Reply:
I don't think a feedback class is needed right now, as this file expands and view gets refactored ill make this a class